lib/puppet/parser/functions/mco_array_to_string.rb

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env ruby
 # vim: set sw=2 sts=2 et tw=80 :
-Puppet::Parser::Functions.newfunction(:mco_array_to_string, type: :rvalue) do |args|
+Puppet::Parser::Functions.newfunction(:mco_array_to_string, :type => :rvalue) do |args|
```

Comment:
Yeah, I myself have to make this change when I clone the module.....

But we as devs can't commit to this. If you can remove this commit it's merge-able.